src/components/ReplyToComment/ReplyToComment.scss

```diff
@@ -0,0 +1,17 @@
+.commentReplyTextarea {
+	background-color: rgba(0, 0, 0, 0.03);
+	border-color: #172439;
+	border-radius: 2px;
+}
+
+.commentReplyButtn {
```

Comment:
@ezrogha change px to rem

Reply:
Thanks for the feedback @dannylwe,
It has been implemented